### PR TITLE
Sys 3213 read me version 0.8.0

### PR DESCRIPTION
**Pull request by micaelffrancoAV**
What: Readme updated to version 0.8.0.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,12 +31,22 @@ const AVN_GATEWAY_URL = 'https://...';
...
+// The AvN address of the payer you will be using:
+const PAYER = '5G7B3...';
+
 // The Ethereum address of the Authority required for minting NFTs, as supplied by Aventus:
 const AVN_AUTHORITY = '0xD3372...';
 
 async function main() {
   const options = { suri: '0x816ef9f2c7f9e8c013fd5fca220a1bf23ff2f3b268f8bcd94d4b5df96534173f'};
+  // For split fee functionality we must specify the payer in the options object.
+  const splitFeeOptions = { suri: '0x816ef9f2c7f9e8c013fd5fca220a1bf23ff2f3b268f8bcd94d4b5df96534173f', payerAddress: PAYER };
```
Comment (nahuseyoum) [inline README.md:43]: I think we should leave the `payerAddress` as null or omit it to show that its optional.

**Review by nahuseyoum — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (micaelffrancoAV) [reply README.md:43]: That is what I am trying to show on line 45: `const defaultSplitFeeOptions = { suri: '0x816ef9f2c7f9e8c013fd5fca220a1bf23ff2f3b268f8bcd94d4b5df96534173f', hasPayer: true };`

Should I change it?

File: README.md [hunk 2]
```diff
@@ -31,12 +31,22 @@ const AVN_GATEWAY_URL = 'https://...';
...
 
+// The AvN address of the payer you will be using:
+const PAYER = '5G7B3...';
+
 // The Ethereum address of the Authority required for minting NFTs, as supplied by Aventus:
 const AVN_AUTHORITY = '0xD3372...';
 
 async function main() {
   const options = { suri: '0x816ef9f2c7f9e8c013fd5fca220a1bf23ff2f3b268f8bcd94d4b5df96534173f'};
+  // For split fee functionality we must specify the payer in the options object.
```
Comment (nahuseyoum) [inline README.md:42]:
```suggestion
  // For split fee functionality we can specify the payer in the options object.
```

File: README.md (hunk 1 above)
Comment (nahuseyoum) [reply README.md:43]: I see, I added a small change to the comments but we can leave this as is.

**Review by nahuseyoum — APPROVED** [review]

**Review by dr-ngubo — APPROVED** [review]

**Pull request merged**